ceph.spec.in

```diff
@@ -134,6 +134,9 @@ BuildRequires:	snappy-devel
 BuildRequires:	udev
 BuildRequires:	util-linux
 BuildRequires:	valgrind-devel
+%if %{with make_check}
+BuildRequires:	which
+%endif
```

Comment:
Honestly, I would probably be happiest if we got rid of which altogether and used type -P instead. It is part of bash and is even more safe as it does a direct lookup in PATH (is not alias-aware, etc).